attention_lens/model/get_model.py

```diff
@@ -1,32 +1,60 @@
 import torch.types
 
 from typing import Union
-from transformer_lens import (
-    HookedTransformer,
-    HookedTransformerConfig,
-    FactoredMatrix,
-    ActivationCache,
-)
+from transformers import AutoModelForCausalLM, AutoConfig, AutoTokenizer
+# from transformer_lens import (
```

Comment:
If we're not using these dependencies then no need to keep dead code, lets remove these comments :)